app/components/widgets/forms/PasswordInstructions.scss

```diff
@@ -1,42 +1,18 @@
-.passwordInstructionsClassic {
+.component {
   color: var(--theme-instructions-text-color);
   font-family: var(--font-light);
   line-height: 1.38;
   margin-top: 20px;
+}
+
+:global(.YoroiClassic) .component {
   opacity: 0.5;
 }
 
-.passwordInstructions {
-  color: var(--theme-instructions-text-color);
+:global(.YoroiModern) .component {
   font-family: var(--font-regular);
   line-height: 1.71;
   margin-top: 14px;
   font-size: 14px;
   letter-spacing: 0;
-
-  ul {
```

Comment:
These were unused, as this component only returns `<p>`
Confirmed with @vantuz-subhuman